packages/jasmine/src/adapter/JasmineConfig.ts

```diff
@@ -81,4 +81,14 @@ export interface JasmineConfig {
      * @see https://jasmine.github.io/api/edge/jasmine#.DEFAULT_TIMEOUT_INTERVAL
      */
     defaultTimeoutInterval?: number;
+
+    /**
+     * @desc
+     *  A list of additional reporters which will be added to the test runner.<BR>
```

Comment:
Would you like to add an example of configuring ReportPortal here? The one from the PR looks great

```
* @example <caption>Using ReportPortal with Protractor and Jasmine</caption>
*  // protractor.conf.js
*  const AgentJasmine = require('@reportportal/agent-js-jasmine')
*  const agent = new AgentJasmine(require('./reportportalConf'))
*  // ...
*  jasmineNodeOpts: {
*    ...
*    reporters: [ agent.getJasmineReporter() ],
*  }
*
* @example <caption>Using ReportPortal with WebdriverIO and Jasmine</caption>
*  // wdio.conf.ts
*  const AgentJasmine = require('@reportportal/agent-js-jasmine')
*  const agent = new AgentJasmine(require('./reportportalConf'))
*  // ...
*  jasmineOpts: {
*    ...
*    reporters: [ agent.getJasmineReporter() ],
*  }
```

To see how those examples would render on the website, run the following commands in the root directory of the project:
```
npm ci
make clean compile site
cd documentation/website
npm start
```

Reply:
examples added!